### ISSUE TITLE
OrientDB.RequestError: Error on unmarshalling content. Class: s

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#5096]
Orientjs and Orientdb 2.1.3
Has worked in 2.0.14

```
begin
LET t0 = SELECT max($depth) as InsertedAtDepth, @rid, * FROM ( TRAVERSE out('BroughtIntoLeft') FROM (SELECT FROM Root WHERE Name = 'Peter') ) WHERE @class = 'Node' or @class = 'Root'
LET t1 = create vertex Node cluster DE_business content {"DisplayName":"Dustin","Email":"dustin@local","Points":0,"Active":false,"Created":1444510064600}
LET t2 = create edge BroughtIntoLeft cluster DE_broughtIntoLeft from $t0 to $t1
```

``` js
db.query(query, {
        class: 's'
    })
```

```
OrientDB.RequestError: Error on unmarshalling content. Class: s
    at Operation.parseError (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\protocol28\operation.js:879:13)
    at Operation.consume (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\protocol28\operation.js:469:35)
    at Connection.process (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\connection.js:380:17)
    at Connection.handleSocketData (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\connection.js:281:17)
    at Socket.emit (events.js:107:17)
    at readableAddChunk (_stream_readable.js:163:16)
    at Socket.Readable.push (_stream_readable.js:126:10)
    at TCP.onread (net.js:538:20)
From previous event:
    at Connection._sendOp (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\connection.js:81:10)
    at Connection.send (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\connection.js:65:17)
    at BinaryTransport.send (E:\Repositorys\test-api\api\node_modules\orientjs\lib\transport\binary\index.js:191:43)
    at Server.send (E:\Repositorys\test-api\api\node_modules\orientjs\lib\server\index.js:112:25)
    at Db.<anonymous> (E:\Repositorys\test-api\api\node_modules\orientjs\lib\db\index.js:152:24)
From previous event:
    at Db.send (E:\Repositorys\test-api\api\node_modules\orientjs\lib\db\index.js:144:4)
    at Db.exec (E:\Repositorys\test-api\api\node_modules\orientjs\lib\db\index.js:247:22)
    at Db.query (E:\Repositorys\test-api\api\node_modules\orientjs\lib\db\index.js:279:15)
    at Object.internals.query (E:\Repositorys\test-api\api\lib-shared\plugins\graph\lib\index.js:333:18)
    at Transaction.all (E:\Repositorys\test-api\api\lib-shared\plugins\graph\lib\index.js:173:15)
    at Object.exports.createNodeOnSide (E:\Repositorys\test-api\api\lib-shared\plugins\graph\lib\index.js:1352:38)
    at E:\Repositorys\test-api\createtestenv.js:48:9
From previous event:
    at Object.exports.createTree (E:\Repositorys\test-api\createtestenv.js:47:5)
    at E:\Repositorys\test-api\createtestenv.js:214:11
    at processImmediate [as _immediateCallback] (timers.js:367:17)
```

**Comment by alpeshgaglani** [comment]
I had the same issue - Luca fixed it in the latest orientdb (2.1.4)

**Comment by StarpTech** [comment]
@alpeshgaglani thanks for the hint.

**Comment by maggiolo00** [comment]
I'm closing this since seems solved by @lvca from ODB 2.1.4
If not please reopen it

Thanks
